### PR TITLE
KAFKA-17390:Remove broker-list in GetOffsetShell

### PR DESCRIPTION
**Pull request by TaiJuWu**
Jira: https://issues.apache.org/jira/browse/KAFKA-17390

*More detailed description of your change,
Remove broker-list in GetOffsetShell

*Summary of testing strategy (including rationale)
for the feature or bug fix. Unit and/or integration
tests are expected for any behaviour change and
system tests should be considered for larger changes.*

### Committer Checklist (excluded from commit message)
- [ ] Verify design and implementation 
- [ ] Verify test coverage and CI build status
- [ ] Verify documentation (including upgrade notes)

### CODE REVIEW

File: tools/src/main/java/org/apache/kafka/tools/GetOffsetShell.java [hunk 1]
```diff
@@ -107,12 +107,7 @@ private static class GetOffsetShellOptions extends CommandDefaultOptions {
         public GetOffsetShellOptions(String[] args) throws TerseException {
             super(args);
 
-            OptionSpec<String> brokerListOpt = parser.accepts("broker-list", "DEPRECATED, use --bootstrap-server instead; ignored if --bootstrap-server is specified. The server(s) to connect to in the form HOST1:PORT1,HOST2:PORT2.")
-                    .withRequiredArg()
-                    .describedAs("HOST1:PORT1,...,HOST3:PORT3")
-                    .ofType(String.class);
             OptionSpec<String> bootstrapServerOpt = parser.accepts("bootstrap-server", "REQUIRED. The server(s) to connect to in the form HOST1:PORT1,HOST2:PORT2.")
```
Comment (chia7712) [inline tools/src/main/java/org/apache/kafka/tools/GetOffsetShell.java]: Please rename `effectiveBrokerListOpt` to `bootstrapServerOpt`. Also, please initialize it in line#110

**Review by chia7712 — COMMENTED** [review]
@TaiJuWu thanks for this patch

File: tools/src/main/java/org/apache/kafka/tools/GetOffsetShell.java (hunk 1 above)
Comment (TaiJuWu) [reply tools/src/main/java/org/apache/kafka/tools/GetOffsetShell.java]: @chia7712 Thanks for review, updated.

**Review by chia7712 — APPROVED** [review]
LGTM

**Pull request merged**